src/sniffer/pkg/config/config.go

```diff
@@ -16,6 +16,8 @@ const (
 	PacketsBufferLengthDefault         = 4096
 	HostsMappingRefreshIntervalKey     = "hosts-mapping-refresh-interval"
 	HostsMappingRefreshIntervalDefault = 500 * time.Millisecond
+	UseExtendedProcfsResolutionKey     = "use-extended-procfs-resolution"
```

Comment:
please open a task for deprecating this flag so we won't forget

Reply:
https://www.notion.so/otterize/Network-Mapper-remove-flag-use-extended-procfs-resolution-11e03ea9c40080b6b5b3ef4f5cb00cd1?pvs=4